dashboards/Home_Dashboard.json

```diff
@@ -1160,7 +1160,7 @@
             "tableColumn": "",
             "targets": [
                 {
-                    "expr": "count(node_cpu{instance=~\"$host\",mode=\"user\"} or rdsosmetrics_General_numVCPUs{instance=~\"$host\"})",
+                    "expr": "count(node_cpu{instance=~\"$host\",mode=\"user\"}) or rdsosmetrics_General_numVCPUs{instance=~\"$host\"}",
```

Comment:
IMHO changes have to be reverted. In this variant we will ignore RDS CPUs

Reply:
Why?

Reply:
Both variants are wrong. 😆 It should be
```
count(node_cpu{instance=~"$host",mode="user"}) + sum(rdsosmetrics_General_numVCPUs{instance=~"$host"})
```

I will send a patch.

Reply:
… plus it needs to be adjusted for the case when no RDS instances are monitored and `sum` returns no data

Reply:
… and this is wrong too because `node_cpu` does not exist for RDS hosts

Reply:
It works ...
![screenshot_20181004_164501](https://user-images.githubusercontent.com/25184101/46478112-ebfd6500-c7f4-11e8-81d0-2dc4aa98063a.png)